apps/src/javalab/ControlButtons.jsx

```diff
@@ -66,16 +66,14 @@ export default function ControlButtons({
       </div>
       <div className={style.rightButtons}>
         <JavalabSettings />
-        {!isEditingStartSources && (
-          <JavalabButton
-            text={finishButtonText}
-            onClick={isSubmittable ? null : onContinue}
-            className={style.buttonBlue}
-            isDisabled={disableFinishButton}
-            id={finishButtonId}
-            tooltipText={finishButtonTooltipText}
-          />
-        )}
+        <JavalabButton
+          text={finishButtonText}
+          onClick={isSubmittable ? null : onContinue}
+          className={style.buttonBlue}
+          isDisabled={disableFinishButton}
```

Comment:
should we have the button disabled if we are in start mode? Not sure what clicking the button does if we are in start mode